CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Can you remove this file? We have a global [Code of Conduct](https://github.com/pulumiverse/.github/blob/main/CODE_OF_CONDUCT.md) file in the [special `.github` organization repository](https://github.com/pulumiverse/.github/blob/main/CODE_OF_CONDUCT.md). The Github documentation doesn't have a clear overview of what can go into the special `.github` repository, but [I found it on StackOverflow](https://stackoverflow.com/a/61301254/7269988).

This global file automatically shows up in the sidebar on the right of every repository. Having a single one promotes consistency over all the repositories.

Reply:
fixed!